components/UnifiedDocFeed/UnifiedDocFeedContainer.tsx

```diff
@@ -248,7 +252,7 @@ function UnifiedDocFeedContainer({
           />
         </div>
       </div>
-      {Boolean(unifiedDocsLoading) ? (
+      {unifiedDocsLoading || isServer() ? (
```

Comment:
Do you still need the `unifiedDocsLoading` given that you have an `isServer()` check now?

Reply:
Yeah.